src/common/logger.ts

```diff
@@ -27,8 +27,8 @@ class Log {
 			case LogLevel.Off:
 				return;
 			case LogLevel.Debug:
-				const hrtime = process.hrtime();
-				const timeStamp = `${hrtime[0]}s ${Math.floor(hrtime[1] / 1000000)}ms`;
+				const hrtime = new Date().getTime()/1000;

```

Comment:
@RMacfarlane I'm not sure if this is the best replacement.

Reply:
I think this is ok! Measuring in milliseconds seems good enough